ivo-projects/todo/src/components/App.js

```diff
@@ -2,28 +2,24 @@ import React, { Component } from 'react';
 import moment from 'moment-timezone';
 import { connect } from 'react-redux';
 import RemindersList from './Reminders';
-import { addReminder, deleteReminder, clearReminders } from '../actions';
+import { addReminder, deleteReminder, clearReminders, editReminder, toggleIsEditingRow } from '../actions';
 
 
 class App extends Component {
     constructor(props) {
         super(props);
         this.state = {
-            disabled: true
+            disabled: true,
```

Comment:
What are these used for?

Reply:
set the state of button's disabled property..the button is disabled if the input field is empty..you cannot add an empty remainder

Reply:
You don't need it. You can have the button disabled if text === ''

Reply:
Delete it and also the editItem property